conans/test/unittests/client/conan_output_test.py

```diff
@@ -51,3 +51,13 @@ def test_output_forced_but_conan_logger():
 
                 assert out.color is False
                 init.assert_not_called()
+
+
+def test_output_chainable():
+    try:
+        ConanOutput(scope="My package")\
+            .title("My title")\
+            .highlight("Worked")\
+            .info("But there was more that needed to be said")
+    except Exception as e:
+        assert False, f"Chained ConanOutput write methods raised an exception {e}"
```

Comment:
In my opinion, it could be easier to check the output itself from any existing test. I'd recommend changing some existing integration ones that use the `self.output` attribute several times.

Let me suggest this one for instance:

*conans/test/integration/py_requires/python_requires_test.py:test_multiple_requires_error*

```python
                def build(self):
                    self.output.info("PKG1 N: %s" % self.python_requires["pkg1"].conanfile.name)\
                               .info("PKG1 V: %s" % self.python_requires["pkg1"].conanfile.version)\
                               .info("PKG1 U: %s" % self.python_requires["pkg1"].conanfile.user)\
                               .info("PKG1 C: %s" % self.python_requires["pkg1"].conanfile.channel)\
                               .info("PKG1 : %s" % self.python_requires["pkg1"].module.myvar)\
                               .info("PKG2 : %s" % self.python_requires["pkg2"].module.myvar)\
                               .info("PKG1F : %s" % self.python_requires["pkg1"].module.myfunct())\
                               .info("PKG2F : %s" % self.python_requires["pkg2"].module.myfunct())
```
What do you think? I believe that changing that one should be good enough.

Reply:
The more tests the merrier, yes! I'll add the one you suggested, thanks :)

Reply:
Done, thanks for your review!